webcompat/views.py

```diff
@@ -306,3 +321,13 @@ def contributors():
 def cssfixme():
     '''Route for CSS Fix me tool'''
     return render_template('cssfixme.html')
+
+
+@app.route('/csp-report', methods=['POST'])
+def log_report():
+    '''Route to record CSP header violations.'''
+    if 'application/csp-report' not in request.headers.get('content-type'):
+        return ('Wrong Content-Type.', 400)
+    with open(app.config['CSP_REPORTS_LOG'], 'a') as r:
+        r.write(request.data + '\n')
```

Comment:
@karlcow one question:

Currently the default `file_handler` is set to `logging.INFO` level, which means anything `INFO` and above will get logged. For CSP, probably something like `logging.WARNING` is appropriate... but that means CSP logs will pollute the regular `webcompat.log`. Which I think we don't want.

The current dumb approach writes to a its own log file, so it avoids this problem.

Is there something obvious I'm missing? Admittedly, the logging module is confusing. 😕